integration-tests/src/test/java/gov/uk/address/api/stepdefinitions/AddressSteps.java

```diff
@@ -75,7 +75,7 @@ public AddressSteps(
         this.addressContext = new AddressContext();
     }
 
-    @When("the user performs a postcode lookup for post code {string}")
+    @And("the user performs a postcode lookup for post code {string}")
```

Comment:
Why did we change this and what impact is it going to have on other tests that use `when`?